src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -12,7 +12,8 @@ object MockWorldState {
 case class MockWorldState(
   accounts: Map[Address, Account] = Map(),
   codeRepo: Map[Address, ByteString] = Map(),
-  storages: Map[Address, MockStorage] = Map()
+  storages: Map[Address, MockStorage] = Map(),
+  headers: Map[BigInt, ByteString] = Map()
```

Comment:
Name is inaccurate, but why do this anyway? `getBlockhash` could simply return a deterministic hash based on its arg, e.g.:

```
def getBlockHash(number: BigInt): Option[ByteString] = 
  Some(ByteString(kec256(number.toByteArray)))
```